### PR TITLE
[PhpUnitBridge] Added missed phpdocs to the PhpUnit Bridge

### PR DESCRIPTION
**Pull request by mxnr**
| Q | A |
| --- | --- |
| Branch? | "master" |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR | Added missed phpdocs ti the PhpUnit Bridge |

### CODE REVIEW

File: src/Symfony/Bridge/PhpUnit/SymfonyTestsListener.php [hunk 1]
```diff
@@ -154,6 +165,10 @@ public function startTest(\PHPUnit_Framework_Test $test)
         }
     }
 
+    /**
+     * @param \PHPUnit_Framework_Test $test
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/PhpUnit/SymfonyTestsListener.php]: We dont add doc blocks that only repeat what the signature already says, the all should be removed.

File: src/Symfony/Bridge/PhpUnit/SymfonyTestsListener.php [hunk 2]
```diff
@@ -154,6 +165,10 @@ public function startTest(\PHPUnit_Framework_Test $test)
...
+     * @param $time
```
Comment (fabpot) [inline src/Symfony/Bridge/PhpUnit/SymfonyTestsListener.php]: This one is wrong as there is no type hint. As a general rule, we never add a `@param` if it does not add value. So, a type hint or a comment if the variable is not clear enough.

File: src/Symfony/Bridge/PhpUnit/ClockMock.php [hunk 3]
```diff
@@ -16,8 +16,16 @@
...
 {
+    /**
+     * @var float
+     */
     private static $now;
 
+    /**
+     * @param null|int $enable
+     *
+     * @return bool
```
Comment (fabpot) [inline src/Symfony/Bridge/PhpUnit/ClockMock.php]: This one is wrong as it can return null as well

File: src/Symfony/Bridge/PhpUnit/ClockMock.php [hunk 4]
```diff
@@ -27,6 +35,11 @@ public static function withClockMock($enable = null)
         self::$now = is_numeric($enable) ? (float) $enable : ($enable ? microtime(true) : null);
     }
 
+    /**
+     * Return current Unix timestamp.
```
Comment (fabpot) [inline src/Symfony/Bridge/PhpUnit/ClockMock.php]: We are using `Returns` everywhere in phpdocs for description, so this needs to be changes everywhere here in this PR.

File: src/Symfony/Bridge/PhpUnit/DnsMock.php [hunk 5]
```diff
@@ -16,7 +16,14 @@
...
 class DnsMock
 {
+    /**
+     * @var array
+     */
     private static $hosts = array();
+
+    /**
+     * @var array
+     */
```
Comment (fabpot) [inline src/Symfony/Bridge/PhpUnit/DnsMock.php]: We don't document private vars, except when it helps IDEs.

File: src/Symfony/Bridge/PhpUnit/ClockMock.php [hunk 6]
```diff
@@ -36,6 +49,13 @@ public static function time()
         return (int) self::$now;
     }
 
+    /**
+     * Delay execution in seconds.
+     *
+     * @param int $s Halt time in seconds.
+     *
+     * @return int
+     */
```
Comment (fabpot) [inline src/Symfony/Bridge/PhpUnit/ClockMock.php]: All phpdocs on mocked methods should be removed IMO.

**Comment by fabpot** [comment]
I think it's not worth it as it is.

**Comment by nicolas-grekas** [comment]
:-1: also for me, closing

**Pull request closed without merging**